charts/landscaper/values.yaml

```diff
@@ -59,6 +59,15 @@ image: {}
 #  tag: ""
 
 controller:
+  # Allows to target a different cluster than the hosting cluster of the controller instance.
+  # Provide the kubeconfig to access the remote Landscaper cluster here (inline or via secretRef).
+  # When providing a secretRef, see ./templates/cluster-kubeconfig-secret.yaml for the correct secret format.
+  # If no value is provided at all, the controller will default to the in-cluster kubeconfig.
+  clusterKubeconfig: {}
```

Comment:
not sure if we should rather name it something like `landscaperKubeconfig` or `lsKubeconfig`.
only `clusterKubeconfig` might be confusing

Reply:
done